kiwi/tasks/system_build.py

```diff
@@ -165,6 +170,14 @@ def process(self):                                      # noqa: C901
                 self.command_args['--set-container-tag']
             )
 
+        if self.command_args['--add-container-label']:
+            for add_label in self.command_args['--add-container-label']:
+                if '=' in add_label:
+                    label_value_pair = add_label.split('=', maxsplit=1)
+                    self.xml_state.add_container_config_label(
+                        label_value_pair[0], label_value_pair[1]
+                    )
```

Comment:
I think this could be done more explicit. imho it's a must have that the format is ```labelname=value``` Thus I would write the code like this:

```python
try:
    (name, value) = add_label.split('=', maxsplit=1)
    self.xml_state.add_container_label(name, value)
except Exception:
    log.warning(
        'Container label {0} ignored. Invalid format: expected labelname=value'.format(add_label)
    )
```

or even raise an error if in this situation we don't want to continue with the defaults

Reply:
Agree, this code is just better. I did not throw an error because we don't for `--set-container-tag` and I would expect both to behave similar. I'd be open to throw an error here also, I do not have strong opinion regarding that, I just tried to be consistent a similar flag.

Reply:
I think it's fine if we log it but we should avoid a silent ignore